### ISSUE TITLE
`theme-check` attempts to initialise on new window and crashes

### ISSUE THREAD

**Issue by ConduciveMammal** [Shopify/theme-check#264]
When opening a new window of VSCode, `theme-check` attempts to initialize but then crashes due to there being no files to process.

```
no implicit conversion of nil into String
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/config.rb:33:in `initialize'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/config.rb:33:in `new'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/config.rb:33:in `find'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/handler.rb:118:in `config_for_path'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/handler.rb:85:in `in_memory_storage'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/handler.rb:27:in `on_initialize'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/server.rb:96:in `process_request'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/server.rb:36:in `block in listen'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/server.rb:35:in `loop'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server/server.rb:35:in `listen'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/lib/theme_check/language_server.rb:20:in `start'
/usr/local/Cellar/theme-check/0.7.2/gems/theme-check-0.7.2/exe/theme-check-language-server:11:in `<top (required)>'
/usr/local/bin/theme-check-language-server:6:in `load'
/usr/local/bin/theme-check-language-server:6:in `<main>'
[Error - 11:02:14 AM] Connection to server got closed. Server will not be restarted.
```

**Comment by charlespwd** [comment]
Fixed in 0.8.1.

```
brew update
brew upgrade theme-check
```

**Comment by ConduciveMammal** [comment]
Hey @charlespwd, this seems to have cropped up again, unfortunately.

Theme-check version: 1.9.2.
